agent/agentssh/agentssh.go

```diff
@@ -180,21 +180,24 @@ func (s *Server) ConnStats() ConnStats {
 }
 
 func (s *Server) sessionHandler(session ssh.Session) {
+	logger := s.logger.With(slog.F("remote_addr", session.RemoteAddr()), slog.F("local_addr", session.LocalAddr()))
+	logger.Info(session.Context(), "handling ssh session")
```

Comment:
This could be logged after the fact so that we only need one message, handling or closing.

Reply:
For debugging, its helpful to know both the start and finish of the session.

Reply:
That's fair! 👍🏻